apps/storybook/src/stories/onchain-info/TokenChip.mdx

```diff
@@ -0,0 +1,67 @@
+import { Meta, Story, Canvas } from "@storybook/blocks";
+import * as TokenChipStories from "./TokenChip.stories";
+import { TokenChip } from "./TokenChip";
+
+<Meta title="MDX/TokenChip" of={TokenChipStories} />
+
+# TokenChip
+
+The `TokenChip` component is a button that displays the token symbol.
+
+## Example
+
+Displaying Ether(ETH) token symbol.
+
+<Canvas>
+  <Story of={TokenChip.ETHTokenChip} />
+</Canvas>
+
+## How to use
+
+Install the following `shadcn/ui` component(s):
+
+- [Button](https://ui.shadcn.com/docs/components/button)
+
+Copy the following code into your code base. And modify import parts as needed.
+
+```tsx
+import { Button } from "@/components/ui/button"; // modify as needed
+
+export type TokenChipProps = {
+  // optional because native tokens don't have addresses
```

Comment:
Is this comment describing the attributes that exist on the Token type from onchainkit? If so it doesn't seem necessary here.

Reply:
Was thinking if we re expecting any autofetches in the future, if not  i will delete no prob

Reply:
Could you clarify what you meant by autofetches?

Reply:
@metalboyrick saw you pushed a commit to remove the comment on another file. Could you remove it on this file as well?

Reply:
fixed